index.js

```diff
@@ -378,11 +378,13 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = ver || msg || opt || undefined
```

Comment:
@bcoe I absolutely agree with this idea, to be honest, the only reason why I kept it this short was because the code before seemed rather valuing shortness either, I would like to make it more readable as well